src/natcap/invest/sdr/sdr_core.pyx

```diff
@@ -419,8 +419,8 @@ def calculate_sediment_deposition(
         xoff = offset_dict['xoff']
         yoff = offset_dict['yoff']
 
-        LOGGER.info('%.2f%% complete', 100.0 * (
-            (seed_row * seed_col) / float(n_cols*n_rows)))
+        LOGGER.info('Sediment deposition %.2f%% complete', 100.0 * (
+            (xoff * yoff) / float(n_cols*n_rows)))
```

Comment:
It makes sense that this should be `xoff`/`yoff` rather than `seed_row`/`seed_col`, since the logging happens in the loop where the offsets are updated. But how did the old version end up not always increasing?

Reply:
@emlys yeah, the issue is that the `processing_stack` is causing pixels to be visited in a non-linear fashion, with order dependent on the flow.  In cases where we're seeing this incorrect logging, we're pushing a lot of pixels on top of a pixel from an earlier block, working through all those new pixels and then jumping back to an earlier block which is then reflected in the logging.

By contrast, using `xoff`, `yoff`, proceeds incrementally and is a better reflection of progress in the raster.

Reply:
Ah, makes sense!